### DISCUSSION TITLE
Setting Danger in GHActions - tokens

### DISCUSSION THREAD

**Discussion by oliwiamichalak** [danger/danger#1412] [Q&A] [answered]
I'm setting up danger in private repository with GHActions. 

GitHub Actions code snipped:

>     - name: Danger
>         env:
>           GITHUB_TOKEN: ${{ secrets.DANGER_GTITHUB_API_TOKEN }}
>         run: |
>           bundle install
>           bundle exec danger --verbose
> 

I recive an error:
![Screenshot 2023-01-09 at 10 42 35](https://user-images.githubusercontent.com/12326936/211290440-00773706-bb02-4db8-a67e-ac2b6f847b70.png)

`DANGER_GTITHUB_API_TOKEN` is setup as secret in repository. 
What else I need to finish configuration? 
How I can create `DANGER_GITHUB_BEARER_TOKEN`?

**Comment by orta** [comment, marked as answer]
`      GITHUB_TOKEN: ${{ secrets.DANGER_GTITHUB_API_TOKEN }}`

typo ^
